docs/buildah-inspect.md

```diff
@@ -33,5 +33,7 @@ buildah inspect --type container containerID
 
 buildah inspect --type image imageID
 
+buildah inspect --format '{{.OCIv1.Config.Env}}' alpine
```

Comment:
This seems a little random, but I am fine with adding it.

Reply:
I'm happy to change it to something else, I was mainly looking to get a multi-level OCI image field as an example as we'd no format example..